daemon/config/config.go

```diff
@@ -161,6 +168,11 @@ func (cfg *Config) Validate() error {
 		cfg.Runtimes[cfg.DefaultRuntime] = types.Runtime{Path: cfg.DefaultRuntime}
 	}
 
+	// validates cgroup driver
+	if err := validateCgroupDriver(cfg.CgroupDriver); err != nil {
```

Comment:
you should directly use `return validateCgroupDriver(cfg.CgroupDriver)`.
Otherwise it will report linter issue in CI.